src/main/java/frc/robot/subsystems/swerve/SwerveModule.java

```diff
@@ -115,29 +118,32 @@ public SwerveModule(int drivePort, int steerPort) {
 
     /**
      * Gets the current state of the module as a `SwerveModulePosition`.
+     * 
      * @return The state of the module.
      */
     public SwerveModulePosition getState() {
         return new SwerveModulePosition(
-            // driveMotor.getSelectedSensorPosition() * DRIVE_TICKS_TO_METERS,
-            driveEncoder.getPosition(),
-            getAngle()
-        );
+                // driveMotor.getSelectedSensorPosition() * DRIVE_TICKS_TO_METERS,
+                driveEncoder.getPosition(),
+                getAngle());
```

Comment:
Oh you used format file didn't you :( now it'll make multi-line constructors and method invocations look ugly like this and break lines suboptimally